doc/nrf/gs_installing.rst

```diff
@@ -462,129 +462,14 @@ Visual Studio Code extension
-
-   #. Set the GNU ARM Embedded Toolchain directory path to the location where it was downloaded (for example, :file:`c:\\gnuarmemb`).
-   #. Close the :guilabel:`nRF Connect options` window.
-   #. Navigate to :guilabel:`Tools` > :guilabel:`Options` and select the :guilabel:`nRF Connect` tab.
-   #. Set the full path of the Zephyr Base directory to :file:`ncs/zephyr`.
-
-#. Set up executables.
-   The process is different depending on your operating system.
-
-   .. tabs::
-
-      .. group-tab:: Windows
-
-         Make sure the locations of executable tools are added to the :envvar:`PATH` variable, including Python, CMake, Ninja, and DTC.
-         On Windows, SES uses the :envvar:`PATH` variable to find executables if they are not set in SES.
-
-      .. group-tab:: Linux
-
-         Make sure the locations of executable tools are added to the :envvar:`PATH` variable, including Python, CMake, Ninja, and DTC.
-         On Linux, SES uses the :envvar:`PATH` variable to find executables if they are not set in SES.
-
-      .. group-tab:: macOS
-
-         If you start SES on macOS by running the :file:`emStudio` executable file, make sure to complete the following steps:
-
-         1. Specify the path to all executables under :guilabel:`Tools` > :guilabel:`Options` (in the :guilabel:`nRF Connect` tab).
-
-            .. figure:: images/ses_options.png
-               :alt: nRF Connect SDK options in SES on Windows
-
-               nRF Connect SDK options in SES (Windows)
-
-            Use this section to change the SES environment settings later as well.
-
-         #. Specify the path to the west tool as an additional CMake option, replacing *path_to_west* with the path to the west executable (for example, :file:`/usr/local/bin/west`):
-
-            .. parsed-literal::
-               :class: highlight
-
-               -DWEST=\ *path_to_west*
-
-         If you start SES from the command line, it uses the global :envvar:`PATH` variable to find the executables.
-         You do not need to explicitly configure the executables in SES.
-
-         Regardless of how you start SES, if you get an error that a tool or command cannot be found, first make sure that the tool is installed.
-         If it is installed, verify that its path is configured correctly in the SES settings or in the :envvar:`PATH` variable.
-
-   ..
-
-If you want to change the SES environment settings after the initial setup, click :guilabel:`Tools` > :guilabel:`Options` and select the :guilabel:`nRF Connect` tab, as shown on the following screenshot from the Windows installation.
-
-.. _ses_options_figure:
-
-   .. figure:: images/ses_options.png
-      :alt: nRF Connect SDK options in SES on Windows
-
-      nRF Connect SDK options in SES (Windows)
-
-If you want to configure tools that are not listed in the SES options, add them to the :envvar:`PATH` variable.
+For instructions specifically for installing, see `Installing using Visual Studio Code`_.
```

Comment:
I think this line could actually be added to the |vsc_extension_instructions| shortcut at this point. It would work also on gs_programming and ug_thingy91...